README.md

```diff
@@ -277,62 +287,137 @@ signIn.renderEl(
+        // - user (object)
+        // - type (string)
+        // - stepUp (object)
+        //   - url (string)
+        //   - finish (function)
         console.log(res.user);
         console.log('Target resource url: ' + res.stepUp.url);
         res.stepUp.finish();
         return;
-      } else {
-        // If the widget is not configured for OIDC, the response will contain
-        // user metadata and a sessionToken that can be converted to an Okta
-        // session cookie:
+      }
+
+      if (res.next) {
+        // If the response contains a redirect URL, the response object will contain a
+        // function that redirects the user when called.
+        // Properties available on `res`:
+        // - status (string)
+        // - user (object)
+        // - type (string)
+        // - next (function)
         console.log(res.user);
-        res.session.setCookieAndRedirect('https://acme.com/app');
+        res.next();
         return;
       }
 
+      // If the widget is not configured for OIDC, the default authentication
+      // type of SESSION_SSO is implied. The response will contain user
+      // metadata and a sessionToken that can be converted to an Okta
+      // session cookie:
+      // Properties available on `res`:
+      // - status (string)
+      // - user (object)
+      // - type (string)
+      // - session (object)
+      //   - token (string)
+      //   - setCookieAndRedirect (function)
+      console.log(res.user);
+      res.session.setCookieAndRedirect('https://acme.com/app');
+      return;
+
+    }
+
+  },
+
+  function error(err) {
+    // This function is invoked with errors the widget cannot recover from:
+    // Known errors: CONFIG_ERROR, UNSUPPORTED_BROWSER_ERROR
+  }
+);
+```
+
+**OIDC Configuration:**
+
+```javascript
+signIn.renderEl(
+  // Assumes there is an empty element on the page with an id of 'osw-container'
+  {el: '#osw-container'},
 
-      // OIDC response
+  function success(res) {
+    // Handling of other statuses in `res.status` is the same as the non-OIDC configuration
+
+    // The user has successfully completed the authentication flow
+    if (res.status === 'SUCCESS') {
 
       // If the widget is configured for OIDC with a single responseType, the
       // response will be the token.
       // i.e. authParams.responseType = 'id_token':
       console.log(res.claims);
+
+      // Example handling of single responseType:
       signIn.tokenManager.add('my_id_token', res);
 
       // If the widget is configured for OIDC with multiple responseTypes, the
       // response will be an array of tokens:
       // i.e. authParams.responseType = ['id_token', 'token']
-      signIn.tokenManager.add('my_id_token', res[0]);
-      signIn.tokenManager.add('my_access_token', res[1]);
+
+      // Example handling of multiple responseTypes instead:
+      // signIn.tokenManager.add('my_id_token', res[0]);
+      // signIn.tokenManager.add('my_access_token', res[1]);
```

Comment:
Here's one way I'm trying to pack two scenarios into one example while still presenting technically valid code. The user can see both examples and uncomment/comment the relevant code example they will need.

Reply:
I don't think `signIn.tokenManager` actually exists. In the next version, we will be removing/deprecating using `signIn.authClient` as well. It's not the widget's responsibility to store tokens, and it never reads them. We can leave this part a little bit vague I think, or provide links to full samples which show how to put it together with the other SDKs

Reply:
I can leave both commented out as hypothetical SDKs that might exist and let the user write the implementation that works best for them, knowing that `res` or `res[0], res[1]` exist for them to consume.